config/example/ir.yaml

```diff
@@ -17,7 +17,7 @@ morph:
   endpoints: # List of websocket RPC endpoints in sidechain
       - wss://sidechain1.fs.neo.org:30333/ws
       - wss://sidechain2.fs.neo.org:30333/ws
-  validators: # List of hex-encoded 33-byte public keys of sidechain validators to vote for at application startup
+  validators: # List of hex-encoded 33-byte public keys of sidechain validators to vote for at application startup; ignored in local consensus mode
```

Comment:
It shouldn't be ignored. Just can be omitted if we're voting for the same set of nodes (most of the time we are). Otherwise we won't be able to vote for a different set (blockchain standby validators list can't change).

I think it was a little different in the previous patch version.

Reply:
> I think it was a little different in the previous patch version.

Yes, it was.

> It shouldn't be ignored.

Can you, please, describe the desired flow? I start thinking that I'm just guessing what should be the true way.

```go
validators = parsedFromCfg
if localMode {
   if len(validators) == 0 {
        validators = committee
    }
} else {
    if len(validators) == 0 {
        panic
    }
}
```

?

Reply:
Yeah, like that.

Reply:
fixed